.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  push:
```

Comment:
once this is ready to be merge we shoul remember to change this to run on pr only

Reply:
I've changed it to run on pushes to `main` and PRs targeting anything, is this what you meant?

Reply:
yes!